### DISCUSSION TITLE
Issue in generating the output after Setup and running the Client app

### DISCUSSION THREAD

**Discussion by RonakPrasad** [practice404/clothes-virtual-try-on#15] [Q&A] [answered]
I have followed the Setup.pdf provided in the repo and made the necessary changes required to run the instances in Colab as well as the Client Application on the local machine.

But when I run the Client Application and upload the cloth and model images.
The following errors are thrown: 

In Colab: 
![image](https://github.com/practice404/clothes-virtual-try-on/assets/68803976/25902d3e-f027-4397-b1b5-30507338554c)

In Client App:
![image](https://github.com/practice404/clothes-virtual-try-on/assets/68803976/9dfbafbe-e854-4043-9997-17faca6cdd66)

It would mean a lot if anyone could help with this issue. 

Awaiting your response.

Thank you.

**Comment by aayushkakkar26** [comment]
bro can you help me running this code ?///

**Comment by SwayamInSync** [comment, marked as answer]
This is resolved in the latest version of repo
